recipes/sortednp/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "sortednp" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a50b4e26a87ba495a2095f3a8e76eccb4cb844ec8bc6531f19f3d36f4801c4d0
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - numpy >=1.14
```

This should specify an exact numpy version (if at all):
 ```suggestion
     - numpy =1.14
```